AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -186,6 +186,18 @@ + (void)deleteDocumentWithPartition:(NSString *)partition
                                           completionHandler:completionHandler];
 }
 
++ (void)setOfflineMode:(BOOL)offlineMode {
+  @synchronized([MSDataStore sharedInstance]) {
+    [MSDataStore sharedInstance].offlineMode = offlineMode;
```

Comment:
This does not need to be synchronized since the property is `atomic`

Reply:
Yeah.. That's true..